bench/perf_run_py/perf_run.py

```diff
@@ -124,6 +124,37 @@ def detailed_perf_stats(config: PerfConfig, reported_stats: List[PerfStat]):
 
     return as_dict
 
+def configure_for_extra_accuracy(config: PerfConfig, arguments: List[str]):
+    """
+    When the `extra_accuracy` flag is set we want to run each test multiple times to 
+    ensure a more stable result. However, this can take a lot of time for longer tests 
+    so limit them to only a few minutes.
+    """
+
+    new_run_max = 5
+    new_run_time="run_time=120"
+    print("==================")
+    print(f"Extra accuracy flag set. Overriding runmax to {new_run_max} and setting -o {new_run_time}")
+    print("==================")
+    
+    config.run_max = new_run_max
+
+    if(arguments):
+        for (i, arg) in enumerate(arguments):
+            if arg.startswith("-o "):
+                if "run_time=" in arg:
+                    print("Error: Attempting to set `run_time` but it has already been set via the `-args` flag`")
+                    exit(1)
+                else:
+                    arguments[i] = arg + "," + new_run_time
+                    return
+                    
+    # Arguments is of None type - initialise.
```

Comment:
np: Can you move the `# Else there is no '-o' argument yet. Add one.` comment up a few lines?

```
# Else there is no `-o` argument yet. Add one.
if not arguments:
    arguments = []
arguments += [f"-o {new_run_time}"]
```

Reply:
Done here: https://github.com/wiredtiger/wiredtiger/pull/8469/commits/bbff47e0ab9018d913aa175d70dba79d91e3a0f2